Antidote/ContactCellView.swift

```diff
@@ -0,0 +1,150 @@
+//
+//  ContactCellView.swift
+//  Antidote
+//
+//  Created by Yoba on 18/12/2016.
+//  Copyright © 2016 Dmytro Vorobiov. All rights reserved.
+//
+
+import Cocoa
+import SnapKit
+
+/**
+    ContactCellView is used in the ContactsTableViewController and represents a cell with short information about user's contact (or friend): contact's username, last message, avatar.
+ 
+    This is also used for few actions: calling a contact, checking it's full information, call history, etc.
+ */
+
+// MARK: init methods and properties
+
+class ContactCellView: NSTableCellView {
+    var userAvatar: RoundedImageView!
+    var lastSenderAvatar: RoundedImageView!
+    var usernameLabel: LabelView!
+    var lastMessageTextLabel: LabelView!
+    var lastMessageDateLabel: LabelView!
+    var lastSeenLabel: LabelView!
+    
+    fileprivate var divider: NSView!
+    
+    struct Constants {
+        static let cellHeight = CGFloat(68)
+        static let borderWidth = CGFloat(2)
+        static let avatarSize = CGFloat(48)
+        static let avatarCornerRadius = CGFloat(24)
+        static let lastSenderAvatarSize = CGFloat(32)
+        static let lastSenderAvatarCornerRadius = CGFloat(16)
+    }
+    
+    override var backgroundStyle: NSBackgroundStyle {
+        didSet {
+            backgroundStyle == .light ? setColorsForLightStyle() : setColorsForDarkStyle()
+        }
+    }
+    
+    required init?(coder: NSCoder) {
+        super.init(coder: coder)
+    }
+    
+    override init(frame: NSRect) {
+        super.init(frame: frame)
+        
+        createSubviews()
+        setupConstraints()
+    }
+}
+
+// MARK: subviews handling
+
+fileprivate extension ContactCellView {
+    func setColorsForLightStyle() {
+        usernameLabel.textColor = NSColor.black
+        lastSenderAvatar.layer?.borderColor = NSColor.green.cgColor
+        lastMessageTextLabel.textColor = BaseConstants.lightGrayColor
+        lastMessageDateLabel.textColor = BaseConstants.lightGrayColor
+        lastSeenLabel.textColor = BaseConstants.lightGrayColor
+        divider.layer?.backgroundColor = BaseConstants.lightGrayColor.cgColor
```

Comment:
It would be a good idea to keep all colors in single place (like BaseConstants). That way they can be changed at any time, also it would be easier to reuse them.